frontend/src/metabase/core/components/TabRow/TabRow.styled.tsx

```diff
@@ -10,6 +10,7 @@ export const TabList = styled(BaseTabList)`
 
   ${BaseTabList.Content} {
     display: flex;
+    overflow: hidden;
```

Comment:
Needed to prevent the tabs from escaping their parent element when one of them has a really long name. Right now the overflowed tabs are not visible, but in a later PR I will reintroduce the scrolling functionality that the old (now deleted) version of this component had.